src/bindings/python/src/openvino/runtime/opset6/ops.py

```diff
@@ -171,19 +161,46 @@ def _(variable_id: str,
     :param name:         Optional name for output node.
     :return: ReadValue node
     """
-    attr_map: Dict[str, Any] = {"variable_id": variable_id}
+    info = VariableInfo()
+    info.variable_id = variable_id
 
     if variable_type is not None:
         if not isinstance(variable_type, Type) and not isinstance(variable_type, str):
-            attr_map["variable_type"] = get_element_type_str(variable_type)
+            info.data_type = get_element_type(variable_type)
         else:
-            attr_map["variable_type"] = variable_type
+            info.data_type = variable_type
 
     if variable_shape is not None:
-        attr_map["variable_shape"] = PartialShape(variable_shape)
+        info.data_shape = PartialShape(variable_shape)
+
+    var_from_info = Variable(info)
+
+    return _read_value(var_from_info)
```

Comment:
Just to ensure, as the Variable is created within this python function scope, then passed to the constructor `ReadValue(const std::shared_ptr<util::Variable>& variable);`, is the lifetime of the `var_from_info` somehow assured?